webrtc/pending-promise-after-close.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>Test pending promises behavior when RTCPeerConnection object is closed</title>
+</head>
+<body>
+  <!-- These files are in place when executing on W3C. -->
```

Comment:
This is an odd comment in some of the existing tests. I'd suggest not copying it around, and removing it where it exists if you like.